test/extended/clusterlogging/deploy_logging.go

```diff
@@ -0,0 +1,124 @@
+package clusterlogging
+
+import (
+	"fmt"
+	"strconv"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var (
+	//CLO resource files to create subscription for cluster-logging-operator
+	CLO = OperatorObjects{
```

Comment:
A more representative test would be to create these resources by way of OLM or what OLM creates.  This is what we are being encouraged to do in our own CI